code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -214,15 +226,16 @@ var/list/mob_hat_cache = list()
 		return
 
 	if(emagged)
-		to_chat(user, "<span class='danger'>\The [user] attempts to load subversive software into you, but your hacked subroutines ignore the attempt.</span>")
+		to_chat(src, "<span class='danger'>\The [user] attempts to load subversive software into you, but your hacked subroutines ignore the attempt.</span>")
 		to_chat(user, "<span class='danger'>You attempt to subvert [src], but the sequencer has no effect.</span>")
 		return
 
-	to_chat(user, "<span class='danger'>You swipe the sequencer across [src]'s interface and watch its eyes flicker.</span>")
```

Comment:
I don't think this line should have been removed, it is the line that that prints to the *user* of the emag.